### PR TITLE
Arctic documentation

### PR DESCRIPTION
**Pull request by bmoscon**
A work in progress on the documentation. I think for the most part, the high level intro is sufficient, and all that needs to be done is:

* minor updates for Chunkstore. Its current docs are based on an older version. Only a few minor features have been introduced, but these should be added and documented
* need documentation for versionstore
* need documentation for tickstore

### CODE REVIEW

**Comment by bmoscon** [comment]
@jamesblackburn I'd like to get this merged in since i think what i have is 1) better than what we currently have and 2) I'll probably only be able to do about 1/2 of what is required for tickstore and about 3/4 of what is required for version store, simply because i do not have as much experience with them. I'd like to merge this in as the "base" and then submit separate PRs for the other 3 remaining items. I can do 1 of them solo, but I'd probably create branches for the other 2 and then once i have a shell/outline request some input from members of the AHL data team.

**Comment by bmoscon** [comment]
@jamesblackburn any problems with me merging this in?

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# Arctic Documentation
+
+Arctic is a tickstore database that sits atop MongoDB. Arctic supports serialization of a number of datatypes for storage in the mongo document model.
```
Comment (jamesblackburn) [inline docs/README.md]: tickstore => timeseries / dataframe ?

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+`initialize_library` has an optional named arg, `lib_type` that defaults to `VersionStore` (more on Arctic storage engine types later).
+
+Once a library is initialized, you can access it like so:
+
+```
+>>> lib = conn['library_name']
+```
+
+Wtih this handle to the library, we can begin to store and retrieve data from Arctic.
```
Comment (jamesblackburn) [inline docs/README.md]: *With

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+(note, most of the storage engines support the same basic methods (read, write, etc), but each have their own set of unique methods as well)
```
Comment (jamesblackburn) [inline docs/README.md]: `read`, `write`

File: docs/README.md (hunk 1 above)
Comment (bmoscon) [reply docs/README.md]: good point!

File: docs/README.md (hunk 2 above)
Comment (bmoscon) [reply docs/README.md]: :+1:

File: docs/README.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+Arctic is a timeseries / dataframe database that sits atop MongoDB. Arctic supports serialization of a number of datatypes for storage in the mongo document model.
+
+Arctic provides a [wrapper](https://github.com/manahl/arctic/blob/docs/arctic/arctic.py) for handling connections to Mongo. The `Arctic` class is what actually connects to Arctic. 
+
+```
+>>> conn = Arctic('127.0.0.1')
+```
+
+There are a number of operations that are possible with just this connection handle. The most basic ones are `list_libraries` and `initiailize_library`.
```
Comment (Akkarinn) [inline docs/README.md]: Note for futur updates, there is a tiny typo: initialize_library

**Pull request merged**